src/CommonLib/aspnetcoreconfig.h

```diff
@@ -215,18 +215,43 @@ class ASPNETCORE_CONFIG : IHttpStoredContext
         return &m_struConfigPath;
     }
 
-    STRU*
-    QueryHostfxrPath()
+    PCWSTR*
+    QueryHostFxrArguments(
+        VOID
+    )
     {
-        return &m_struHostFxrPath;
+        return m_ppStrArguments;
     }
 
-    BOOL
-    QueryIsStandAloneApplication(
+    DWORD
+    QueryHostFxrArgCount(
         VOID
     )
     {
-        return m_fIsStandAloneApplication;
+        return m_dwArgc;
+    }
+
+    STRU*
+    QueryHostFxrFullPath(
+        VOID
+    )
+    {
+        return &m_struHostFxrLocation;
+    }
+
+    VOID
+    SetHostFxrArguments(
+        DWORD dwArgc,
+        PCWSTR* ppStrArguments
+    )
+    {
```

Comment:
it will be nice to add debug assert here indicating the contact between caller and callee